src/pages/iou/steps/IOUParticipantsPage/IOUParticipantsRequest.js

```diff
@@ -71,6 +67,22 @@ class IOUParticipantsRequest extends Component {
         };
     }
 
+    /**
+     * @param {string} searchTerm
+     * @returns {Object}
+     */
+    getRequestOptions(searchTerm = '') {
+        return OptionsListUtils.getNewChatOptions(
+            this.props.reports,
+            this.props.personalDetails,
+            this.props.betas,
+            searchTerm,
+            [],
+            CONST.EXPENSIFY_EMAILS,
+            this.props.iouType !== CONST.IOU.MONEY_REQUEST_TYPE.SEND,
```

Comment:
This would also show the options for Split, right? We dont want that

Reply:
This component doesn't get used in that flow.

https://github.com/Expensify/App/blob/e3597c68d03ad48b50b328daf5cf3dbd269b678d/src/pages/iou/steps/IOUParticipantsPage/IOUParticipantsPage.js#L72-L87

Reply:
Thanks! Havent realized, still better to use the request. Thanks for updating it